doc/classes/OS.xml

```diff
@@ -277,6 +277,16 @@
 				[b]Note:[/b] Thread IDs are not deterministic and may be reused across application restarts.
 			</description>
 		</method>
+		<method name="get_memory_info" qualifiers="const">
+			<return type="Dictionary" />
+			<description>
+				Returns the [Dictionary] with the following keys:
```

Comment:
Yeah I think it should clarify that keys can be missing if the information couldn't be found. But it means users will basically have to do `if meminfo.has("physical"):` before using it, or `meminfo.get("physical", -1)`.

It could make sense to always define the keys with `-1` value for "unknown", so we can make it a clear contract that the dictionary always has these 4 keys.